randovania/games/prime2/generator/hint_distributor.py

```diff
@@ -38,7 +38,7 @@ def is_pickup_interesting(self, target: PickupTarget, player_id: int, hint_node:
                 return False
         if target.player == player_id and ("Translator" in target.pickup.name):
             # don't place a translator hint on its color of lore scan
-            return hint_node.extra.get("translator") not in target.pickup.name
+            return hint_node.requirement_display_name not in target.pickup.name
```

Comment:
I don't like either that you're using something you called display name for core logic.

Reply:
I'll add a FIXME; I never should've used that attribute for this in the first place really